analyzers/src/SonarAnalyzer.CSharp/Rules/GotoStatement.cs

```diff
@@ -28,7 +28,7 @@ namespace SonarAnalyzer.Rules.CSharp
     [DiagnosticAnalyzer(LanguageNames.CSharp)]
     public sealed class GotoStatement : GotoStatementBase<SyntaxKind>
     {
-        public GotoStatement() : base(RspecStrings.ResourceManager) { }
+        protected override ILanguageFacade<SyntaxKind> Language => CSharpFacade.Instance;
 
         protected override SyntaxKind[] GotoSyntaxKinds => new[]
         {
```

Comment:
I intentionally didn't put it there. I doubt it would be reused by any other rule. And it's questionable what the pairing should be: 1x VB vs. all 3x C# or 1x VB vs 1x C# => hard to answer => it's better not to have it in facade.